### ISSUE TITLE
Add racing to the GUI

### ISSUE THREAD

**Issue by ZainabAbualrehi** [gzdwsu/RaiderSMA#69]
I noticed that racing is part of the project but it still not added to the project GUI ,so racing will be added to be an option beside soccer and wrestling.

**Comment by ZainabAbualrehi** [comment]
I, Zainab Abualrehi, will work on this issue in project 3

**Comment by sdavis00255** [comment]
There is also bowling, and I just submitted a pull request for golfing. We just figured with so many people working on a GUI at the same time would cause issues as seen in the project two pull requests.

**Comment by ZainabAbualrehi** [comment]
Yes there were issues in project 2 pull requests and I’m worried that it will happen again this time.
I just finished my project I will try to submit a pull request now.

Regards
Zainab.

On Apr 24, 2020, at 5:40 PM, Steven Davis <notifications@github.com> wrote:

﻿

There is also bowling, and I just submitted a pull request for golfing. We just figured with so many people working on a GUI at the same time would cause issues as seen in the project two pull requests.
